packages/flutter/lib/src/widgets/animated_size.dart

```diff
@@ -32,11 +32,6 @@ class AnimatedSize extends StatefulWidget {
     this.curve = Curves.linear,
     required this.duration,
     this.reverseDuration,
-    @Deprecated(
-      'This field is now ignored. '
-      'This feature was deprecated after v2.2.0-10.1.pre.'
-    )
-    TickerProvider? vsync,
```

Comment:
Can you add a `removeParameter` dart fix?

Reply:
Done.